supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -13,7 +13,7 @@
 [discrete]
 [[backtrace]]
 ==== image:images/yes.png[yes] backtrace (noun)
-*Description*: "Backtrace" is another name for "stack trace" (or "stack backtrace"), which is a report of the active stack frames (function calls) at a certain point in time during the execution of a program. The Python programming language uses the term "traceback", possibly because the stack frames are printed in the opposite order of those presented by gdb, the GNU Debugger. "Traceback" is the preferred term when referring to a Python stack trace.
+*Description*: _Backtrace_ is another name for _stack trace_ (or _stack backtrace_), which is a report of the active stack frames (function calls) at a certain point in time during the execution of a program. The Python programming language uses the term _traceback_, possibly because the stack frames are printed in the opposite order of those presented by gdb, the GNU Debugger. _Traceback_ is the preferred term when referring to a Python stack trace.
```

Comment:
I have to say that I'm uncertain whether the two "traceback" instances in the last two sentences should be italics or quotes. I'm leaning toward quotes because we're talking about what term to use, but could use some other thoughts here.

Reply:
Yes, good point. Both sentences do suggest on how to use the term. I updated with quotes.